### ISSUE TITLE
Expected ",", "matrix", "rotate", "scale", "skewX", "skewY", "translate", [ \t\r\n], or end of input but "0" found.

### ISSUE THREAD

**Issue by YeomanYe** [FormidableLabs/victory-native#366]
the issue appear when i use VictoryLabel transform 

```js
<VictoryPie
                        height={245}
                        width={width}
                        padding={0}
                        labelComponent={<VictoryLabel transform={(data)=>{console.log('transform',data);return 'translate(50,50)'}}/>}
                        style={{labels: {
                                fontSize: 15, fill: "rgb(120,120,120)",padding:8,position:'relative',        fontFamily:'PingFang-Regular'}}}
                        radius={70}
                        innerRadius={50}
                        colorScale={chartData.colors}
                        data={chartData.datas}
                    />
```

enviroment:
"react-native": "0.50.3",
"react-native-svg": "^6.5.0",
"react": "16.0.0",
"victory-native": "^30.1.0"

![image](https://user-images.githubusercontent.com/18027354/44295164-5b1b0a80-a2d6-11e8-8c87-f3dc97d67adb.png)

**Comment by boygirl** [comment]
@YeomanYe thank you for reporting this, I can confirm the issue. I think this is likely caused by a change in `react-native-svg`, but I will investigate further, and see if I can find a workaround.

**Comment by boygirl** [comment]
@YeomanYe this should be fixed in `victory-native@30.3.0`
